### PR TITLE
Replace `local-testnet` with Kurtosis.

### PR DESCRIPTION
**Pull request by jimmygchen**
## Issue Addressed

Lighthouse's `local-testnet` scripts and the `lcli` commands it uses are being deprecated and will be removed in [this Lighthouse PR](https://github.com/sigp/lighthouse/pull/5865). 

This PR updates the local testnet scripts to use [kurtosis](https://github.com/kurtosis-tech) instead. 

This should make the testnet a lot easier to maintain, as the `local-testnet` directory now only contains a simple network config file and basic wrapper scripts to start and stop the testnet.

The usage is similar to the old script:
1. Modify `network_params.yaml` if required
2. Start with `./start_local_testnet.sh`
3. Stop with `./stop_local_testnet.sh`

### CODE REVIEW

**Comment by rickimoore** [comment]
super cool :)

File: local-testnet/network_params.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    cl_image: sigp/lighthouse:latest
+    cl_extra_params:
+      - --target-peers=3
+    validator_count: 80
+    count: 4
+network_params:
+  deneb_fork_epoch: 0
+  seconds_per_slot: 3
+  preregistered_validator_keys_mnemonic: "giant issue aisle success illegal bike spike question tent bar rely arctic volcano long crawl hungry vocal artwork sniff fantasy very lucky have athlete"
+global_log_level: info
```
Comment (antondlr) [inline local-testnet/network_params.yaml:18]:
```suggestion
keymanager_enabled: true
global_log_level: info
```
we need to expose the VC API, enabling `keymanager` is the proper way to do so

Comment (jimmygchen) [reply local-testnet/network_params.yaml:18]: Good catch! Forgot about that. Thanks!

File: local-testnet/network_params.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Full configuration reference [here](https://github.com/kurtosis-tech/ethereum-package?tab=readme-ov-file#configuration).
+participants:
+  - el_type: geth
+    el_image: ethereum/client-go:latest
+    cl_type: lighthouse
```
Comment (antondlr) [inline local-testnet/network_params.yaml:5]:
```suggestion
    el_extra_params: 
      - --gui
    cl_type: lighthouse
```

Comment (jimmygchen) [reply local-testnet/network_params.yaml:5]: Oh I've just added that to `cl_extra_params`

**Pull request merged**